src/result_processor.c

```diff
@@ -1032,7 +1071,7 @@ void SetLoadersForMainThread(AREQ *r) {
 static char *RPTypeLookup[RP_MAX] = {"Index",   "Loader",    "Threadsafe-Loader", "Scorer",
                                      "Sorter",  "Counter",   "Pager/Limiter",     "Highlighter",
                                      "Grouper", "Projector", "Filter",            "Profile",
-                                     "Network", "Metrics Applier"};
+                                     "Network", "Metrics Applier", "Key Name Loader"};
```

Comment:
Nitpick: Should `Key Name Loader` have similar naming as `Threadsafe-Loader`? (dash instead of space? Although we also have spaces in `Metrics Applier`)